source/server/admin/logs_handler.cc

```diff
@@ -56,56 +61,106 @@ bool LogsHandler::changeLogLevel(const Http::Utility::QueryParams& params) {
     return false;
   }
 
-  std::string name = params.begin()->first;
-  std::string level = params.begin()->second;
+  const auto& it = params.begin();
```

Comment:
usually I don't see begin() assigned to a reference, but I guess I'm OK with it if ASAN is OK :)